src/main/webapp/add_release.html

```diff
@@ -0,0 +1,53 @@
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+      <title>New Release</title>
+      
+      <link rel="stylesheet" href="css/style.css">
+      <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta/css/bootstrap.min.css" integrity="sha384-/Y6pD6FV/Vv2HJnA6t+vslU6fwYXjCFtcEpHbNJ0lyAFsXTsjBbfaDjzALeQsN6M" crossorigin="anonymous">
+      <script src="https://kit.fontawesome.com/4588e35b97.js" crossorigin="anonymous"></script>    
+  </head>
+
+   <body>
+        <div class="jumbotron">
+            <h1>New Release</h1>  
+        </div>  
+        <div class="row">
+            <div class="leftcolumn">
+                <div class="row">
+                    <div id="sdkleft">
+                        <p>Release Name</p>
+                    </div>
+                </div>
+                <div class="row">
+                    <div id="sdkleft">
+                        <p>Release Manager</p>
+                    </div>
+                </div>
+                <div class="row">
+                    <div id="sdkleft">
+                        <p>Hotlists</p>
+                    </div>
+                </div>
+            </div>
+            <div class="rightcolumn">
+                <div class="row">
+                    <div id="sdklarge">
+                        <p>Cold Freeze:</p>
+                        <p>Launchcal Deadline:</p>
+                        <p>Launch:</p>
```

Comment:
All the fields in this page are supposed to allow inputs from user. But it looks like you have just added the labels, can you please check.